### ISSUE TITLE
Feature Request: Longer history

### ISSUE THREAD

**Issue by mrwee** [thomasnordquist/MQTT-Explorer#321]
Would be nice if the history could be longer, so it's easier to evaluate back in time.

**Comment by 8666** [comment]
Or shorter. I used MQTT Spy before, and there was a limit option.  After that, older values were removed, but at least one/latest value was preserved.

**Comment by christoschronopoulos** [comment]
Indeed, more than 100 values in history would be very useful!! Thanks a lot for your effort!
